testtools/testcase.py

```diff
@@ -497,6 +492,23 @@ def assertThat(self, matchee, matcher, message='', verbose=False):
         if mismatch_error is not None:
             raise mismatch_error
 
+    def assertItemsEqual(self, a, b, message=''):
+        """An unordered sequence specific comparison. It asserts that
+        actual_seq and expected_seq have the same element counts.
+        Equivalent to::
+            self.assertEqual(Counter(iter(actual_seq)),
+                             Counter(iter(expected_seq)))
+        Asserts that each element has the same count in both sequences.
+        Example:
+            - [0, 1, 1] and [1, 0, 1] compare equal.
+            - [0, 0, 1] and [0, 1] compare unequal.
+        """
+        if hasattr(self, 'assertCountEqual'):
+            self.assertCountEqual(a, b, message)
+        else:
+            self.assertEqual(collections.Counter(iter(a)),
+                             collections.Counter(iter(b)))
+
```

Comment:
```suggestion
    if not six.PY2:
        assertItemsEqual = unittest.TestCase.assertCountEqual

```

Reply:
this makes assertCountEqual and assertItemsEqual work on py2 and py3 - I assume that's what you wanted?

Reply:
I think it would be good to try to avoid introducing six usage if possible to make it easier when py2 support is dropped. Which getting rid of unittest2 is a step towards.

Reply:
@stmcginnis updated to use sys.version_info

imho it's best to use six - then we can remove it with pyupgrade

Reply:
True. I'm sure there will be other cleanup work to do.

Reply:
This actually was leftover from a rebase. Testtools master doesn't support python2 anymore the min python version is 3.5, so I need to remove the if else here

Reply:
so you think
```suggestion
    assertItemsEqual = unittest.TestCase.assertCountEqual
```

Reply:
@mtreinish ^